spynl/main/utils.py

```diff
@@ -445,26 +439,44 @@ def send_exception_to_external_monitoring(user_info=None, exc_info=None,
         log.warning('Could not add user info to NewRelic on exception: %s', e)
 
 
-def log_error(exc, request, top_msg, error_type, error_msg):
+def log_error(exc, request, top_msg, error_type=None, error_msg=None):
     """
     Log the error from an error view to the log, and to external monitoring.
     Make sure the __cause__ of the exception is used.
     """
     log = get_logger()
 
-    user_info = get_user_info(request, purpose='error_view')
+    if not error_type:
+        error_type = exc.__class__.__name__
+    if error_type.endswith('Exception'):
+        error_type = error_type[:-len('Exception')]
```

Comment:
I would be ok with removing these two lines, maybe someone else would like to chime in? (removing Exception from the error type) Do we do this for a reason?